script.js

```diff
@@ -0,0 +1,141 @@
+class ProductAPI {

```

Comment:
where is our BaseAPIService class, create BaseAPIService class and extend them in the your product service.

Reply:
done